src/Compilers/VisualBasic/Test/Emit/Emit/CompilationEmitTests.vb

```diff
@@ -2946,5 +2946,34 @@ BC37256: An error occurred while writing the output file: <%= output.ThrownExcep
 </expected>)
             End Using
         End Sub
+
+
+        <Fact>
+        <WorkItem(11691, "https://github.com/dotnet/roslyn/issues/11691")>
+        Public Sub ObsoleteAttributeOverride()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlib45AndVBRuntime(
+                <compilation>
+                    <file name="a.vb">
+                        <![CDATA[
+imports System
+
+Public MustInherit Class BaseClass(of T)
+    Public MustOverride Function Method(input As T) As Integer
+End Class
+    
+Public Class DerivingClass(Of T) Inherits BaseClass(Of T)
+    <Obsolete("Deprecated")>
+    Public Overrides Sub Method(input As T)
+        Throw New NotImplementedException()
+    End Sub
+End Class
+]]>
+                    </file>
+                </compilation>)
+
+            compilation.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](start = 24, length = 17)

Please use AssertTherseDiagnostics helper instead, see previous unit-test for an example.